apps/server/src/shared/repo/dashboard/dashboard.model.mapper.spec.ts

```diff
@@ -62,7 +63,7 @@ describe('dashboard model mapper', () => {
 			const elementId = new ObjectId().toString();
 			const oldElementId = new ObjectId().toString();
 			const newElementId = new ObjectId().toString();
-			const originalDashboard = new DashboardModelEntity(dashboardId);
+			const originalDashboard = new DashboardModelEntity({ id: dashboardId, user: new ObjectId().toString() });
```

Comment:
userId

Reply:
Similarly to the other explanation: user can be lazily populated with the complete user info.